README

```diff
@@ -23,8 +26,8 @@ class reference.
 Easily-installable prebuilt binaries for Windows and macOS are available from
 https://poedit.net/download.
 
-Official binaries for Linux are available as a Snap at https://snapcraft.io/poedit
-Most Linux distributions also include native Poedit packages.
+Official binaries for Linux are available as a Snap package at https://snapcraft.io/poedit
+Most Linux|GNU distributions also include native Poedit packages.
```

Comment:
The actual spelling of that is "GNU/Linux", not this "Linux|GNU" nonsense, and I am not going to engage in this kind of political speech here.

Reply:
That was what I was trying to avoid. All in good fun. Do you want me to remove it?